docs/getting-started/quickstart-tt-debugger.md

```diff
@@ -91,10 +91,15 @@ npx dbos-cloud register -u <username>
 npx dbos-cloud db provision <database-instance-name> -U <database-username>
 ```
 
+:::tip
+Time travel is disabled by default. If you already deployed an application from quickstart, please delete your application by running `npx dbos-cloud app delete`,
```

Comment:
Instead say something more positive like "Make sure you enable time travel when registering your application!"